### PR TITLE
Added ability to automatically run on Repl.it 

### PR DESCRIPTION
**Pull request by Mosrod**

### CODE REVIEW

**Comment by richrd** [comment]
Thanks, seems like a fun way to be able to test out the editor!

File: .replit [hunk 1]
```diff
@@ -0,0 +1,2 @@
+language = "nodejs"
```
[the remark this answers is not in the document]
Comment (Mosrod) [reply .replit]: The correct language should be python3.

**Comment by Mosrod** [comment]
I have fixed the language to python3.

**Comment by richrd** [comment]
Cheers! Thanks for the PR. Merging :+1:

**Pull request merged**